build.gradle

```diff
@@ -36,7 +36,23 @@ ext.wso2SnakeYamlVersion = project.wso2SnakeYamlVersion
 ext.bouncycastleVersion = project.bouncycastleVersion
 ext.guavaVersion = project.guavaVersion
 ext.wso2CarbonMetricsVersion = project.wso2CarbonMetricsVersion
+ext.stdlibHttpVersion = project.stdlibHttpVersion
+ext.stdlibMimeVersion = project.stdlibMimeVersion
+ext.stdlibAuthVersion = project.stdlibAuthVersion
+ext.stdlibFileVersion = project.stdlibFileVersion
+ext.stdlibCacheVersion = project.stdlibCacheVersion
+ext.stdlibCryptoVersion = project.stdlibCryptoVersion
 ext.stdlibEncodingVersion = project.stdlibEncodingVersion
+ext.stdlibEncodingVersion = project.stdlibEncodingVersion
```

Comment:
Removing duplicate
```suggestion
ext.stdlibEncodingVersion = project.stdlibEncodingVersion
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-websub/pull/17/commits/dffc9024569233a20a1e9b4b81cc35ed80ce5024